rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -207,15 +207,15 @@ namespace rsx
 		{
 			if (rsx->in_begin_end)
 			{
-				rsx->append_array_element(arg & 0xFFFF);
-				rsx->append_array_element(arg >> 16);
+				rsx->append_array_element(se_storage<u16>::swap(arg & 0xFFFF));
+				rsx->append_array_element(se_storage<u16>::swap(arg >> 16));
```

Comment:
I'm assuming the indices are already BE before being shifted and ORed
If they were converted to LE -> packed -> result swapped that makes more sense.
The code above widens all indices to u32, not packing them into a byte stream. I wanted to only support one type of index buffer for this mode.